### DISCUSSION TITLE
React native and firebse error:- auth network request failed

### DISCUSSION THREAD

**Discussion by roshan-29** [invertase/react-native-firebase#7590] [Q&A]
Hello..I am doing one food Donation app..using react native and firebase..first it was logging perfectly .But now I am facing login issue also the days fetch issue ..error are like ...firebase auth network request failed and also one more error is : Console Error

Error fetching donated foods: Error: FIRESTORE (10.7.2) INTERNAL ASSERTION FAILED: Unexpected

I am not using reactnative firebase ..But I try to use that it will give error like u using the package which is not present but installed  the package also..my firebase version is 10.7.2. If I change ther version also getting the same errors...Also I am using expo go app..can anyone help me to configure native code .. because I heard that if we use expo then u should set some environment for native firebase..can anyone help me to this
![Screenshot_2024-01-24-19-29-18-77_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/7027e229-4c6f-4ca3-b2b0-047c3a5bf4f9)
![IMG20240124165650](https://github.com/invertase/react-native-firebase/assets/115919772/99267403-b5df-40bb-94ba-7d9e3b7874f6)
![IMG-20240122-WA0028](https://github.com/invertase/react-native-firebase/assets/115919772/5b295b11-d231-4206-9471-c29d59b48f4b)
![Screenshot_2024-01-22-11-46-20-19_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/58274f81-86fd-437f-b512-4cfebdbc6558)
![Screenshot_2024-01-21-15-41-10-78_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/14741382-1981-4890-bf77-6356033631c4)
![Screenshot_2024-01-21-15-39-36-61_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/dcdfd2f6-1cc6-4163-925c-fb5b8fdca5b2)
![Screenshot_2024-01-21-13-33-42-83_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/7348b15e-0a11-4ffb-81e5-d9f64c767c04)
![Screenshot_2024-01-21-13-37-17-60_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/e2df73b3-1aa8-4e32-a106-8662b9b7d7da)
![Screenshot_2024-01-21-11-40-19-00_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/cfcae938-aa5e-4736-ae92-f7cc99a27ed9)
![Screenshot_2024-01-21-11-37-37-62_f73b71075b1de7323614b647fe394240](https://github.com/invertase/react-native-firebase/assets/115919772/f3d42414-c2e8-410e-b134-f10074456deb)
![Uploading Screenshot_2024-01-24-19-29-18-77_f73b71075b1de7323614b647fe394240.jpg…]()

**Comment by mikehardy** [comment]
Expo Go is not compatible with any native modules that are not directly built in to Expo Go, by Expo.
You will need to use firebase-js-sdk, following their documentation, and opening issues with them if you have problems
